### DISCUSSION TITLE
How to use the Cortex remote read API?

### DISCUSSION THREAD

**Discussion by ChinmayMishra** [cortexproject/cortex#3797] [Q&A] [answered]
Please suggest some examples (curl request) for Cortex remote API

**Comment by pracucci** [comment, marked as answer]
As you stated, we're currently lacking any remote read API doc. However, the remote read API is Prometheus compatible ([see doc](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_read)) so I would suggest you to look for any Prometheus remote read documentation.

**Reply by ChinmayMishra** [reply]
Thanks @pracucci for the quick reply. I still could not find any documentation on how to use this API using curl. But I am able to make it work with Go program. Thanks a lot for your support
